easybuild/easyconfigs/l/libgpuarray/libgpuarray-0.7.5-goolfc-2018a.eb

```diff
@@ -0,0 +1,46 @@
+##
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+#
+# Copyright:: Copyright 2012-2016 University of Luxembourg / LCSB, Cyprus Institute / CaSToRC,
+#                                 Ghent University / The Francis Crick Institute
+# Authors::
+# * Ake Sandgren < ake@hpc2n.umu.se>
+# License::   MIT/GPL
+##
+
+name = 'libgpuarray'
```

Comment:
@akesandgren This requires a custom easyblock for `libgpuarray` (https://github.com/easybuilders/easybuild-easyblocks/pull/1316), but there's another approach that doesn't require this yet is fairly clean, see #5429...

Reply:
Ah, hadn't seen that one. Guess you prefer the 5429 way?

Reply:
In this case, yes, I think the easyblock is overkill... But maybe @wpoely86 needs to agree with me first in #5429

Reply:
ping @wpoely86 w.r.t. #5429...